sdk/dataprotection/Azure.ResourceManager.DataProtectionBackup/src/Generated/Models/UnlockDeleteResponse.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.ResourceManager.DataProtectionBackup.Models
+{
+    /// <summary> Response of Unlock Delete API. </summary>
+    public partial class UnlockDeleteResponse
```

Comment:
Consider renaming to `DataProtectionUnlockDeleteResult` by adding below in `rename-mapping` section of autorest.md
```yaml
UnlockDeleteResponse: DataProtectionUnlockDeleteResult
```